### PR TITLE
feat: Yarn PnP support

### LINKED ISSUE #160: Yarn Plug'n'Play support
This extension is really awesome! But, not yet supported by Yarn VS Code SDK. Please make it works with Yarn 2 Plug'n'Play. Thanks!

### PR DESCRIPTION
**Pull request by DanSnow**
Thanks for this extension! It's really useful.

resolved #160

### CODE REVIEW

**Comment by bradlc** [comment]
Wow this is amazing, thanks so much for this @DanSnow! Going to take a look 👍

File: src/class-names/index.js [hunk 1]
```diff
@@ -76,6 +76,8 @@ export default async function getClassNames(cwd = process.cwd(), { onChange = ()
               })
             )
           )
+        } catch (error) {
+          throw error
```
Comment (DanSnow) [inline src/class-names/index.js:80]: `try { ... } finally { ... }` won't swallow the error.

Comment (bradlc) [reply src/class-names/index.js:80]: Oh cool sorry, my bad 👍 Never seen that before. Feel free to change it back

Comment (rizkyramadhan) [reply src/class-names/index.js:80]: This doesn't work on yarn workspaces, but works on root project...

Comment (rizkyramadhan) [reply src/class-names/index.js:80]: Sorry, it does work, but make sure to include: 

```
{ 
purge: ['./pages/**/*.{js,jsx,ts,tsx}'],
}
```

in your tailwind.config.js

**Pull request closed without merging**